aea/mail/base.py

```diff
@@ -413,7 +413,7 @@ def connect(self) -> None:
             if self.connection_status.is_connected:
                 logger.debug("Multiplexer already connected.")
                 return
-            self._start_loop_threaded()
+            self._start_loop_threaded_if_not_running()
```

Comment:
a better name for that method. Only if not running, the loop is started in an ad-hoc thread.